migration/tests/tests.rs

```diff
@@ -0,0 +1,33 @@
+#![cfg(test)]
+
+use migration::{Migrator, MigratorTrait};
+use test_context::test_context;
+use test_log::test;
+use trustify_common::db::{test::TrustifyContext, ConnectionOrTransaction};
+
+#[test_context(TrustifyContext, skip_teardown)]
+#[test(tokio::test)]
+async fn test_migrations(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
+    let db = ctx.db;
+
+    let conn_or_tx = db.connection(&());
+    let conn = match conn_or_tx {
+        ConnectionOrTransaction::Connection(conn) => conn,
+        _ => {
+            panic!("Error while getting db connection");
+        }
+    };
+
+    let db_ref = &db;
+    let migrations = Migrator::get_applied_migrations(db_ref).await?;
+    // 'Migrator.up' was called in bootstrap function when using TrustifyContext.
+    // At this point we already have migrations.
+    assert!(migrations.len() > 1);
+
+    Migrator::refresh(conn).await?;
```

Comment:
...you can just call this:

```suggestion
    db.refresh().await?;
```